src/main/java/com/ncov/module/controller/MaterialsController.java

```diff
@@ -26,35 +27,16 @@
 
     private final UserContext userContext;
     private final MaterialSuppliedService materialSuppliedService;
+    private final MaterialRequiredService materialRequiredService;
 
     @ApiOperation(
             value = "Create new required material.",
             tags = SwaggerConstants.TAG_REQUIRED_MATERIALS
     )
     @PostMapping("/required-materials")
     @ResponseStatus(HttpStatus.CREATED)
```

Comment:
Would be good if this endpoint is limited only to hospitals. See [here](https://github.com/4ncov/service-4ncov/pull/23/files#diff-85352cc4e78ba2dde7b710dbb4c77d69R109)